Description: fix compatibility with newer versions of automake
 Since we want to run dh-autoreconf so that the package is always current
 with respect to portability to new architectures, we also need to make sure
 the Makefile.am works with the latest versions of automake.  This requires
 three fixes:
  - fix configure.ac to not set RESOLVES to ' # ', which causes further
    libraries on the line to be commented out
  - fix duplicate references to a header in zzip/Makefile.am
  - fix missing dependency on install-libLTLIBRARIES in the
    install-exec-local target.
Author: Steve Langasek <steve.langasek@ubuntu.com>

=== modified file 'zzip/Makefile.am'
Index: zziplib/zzip/Makefile.am
===================================================================
--- zziplib.orig/zzip/Makefile.am	2014-01-26 19:42:57.003717930 -0500
+++ zziplib/zzip/Makefile.am	2014-01-26 19:42:56.995717930 -0500
@@ -85,7 +85,7 @@
 	sed -e 's|zzip.h|zzip-io.h|' -e 's|zzip/lib.h|zzip/plugin.h|' \
 	$(DESTDIR)$(includedir)/zzip.h >$(DESTDIR)$(includedir)/zzip-io.h
 
-install-exec-hook:
+install-exec-hook: install-libLTLIBRARIES
 	@ for i in . $(DESTDIR)$(libdir)/libzzip*.so.13 \
         ; do test -d $$i && continue ; test -f $$i || continue \
         ; lib=`basename "$$i" .so.13` \
Index: zziplib/configure.ac
===================================================================
--- zziplib.orig/configure.ac	2014-01-26 19:42:57.003717930 -0500
+++ zziplib/configure.ac	2014-01-26 19:42:56.995717930 -0500
@@ -308,7 +308,7 @@
          # TODO: that is for backward compatibility only
       fi
     fi
-    RESOLVES=" # "
+    RESOLVES=" "
 ;; esac
 AC_MSG_RESULT([$ZZIPLIB_LDFLAGS $RESOLVES])
 AC_SUBST([ZZIPLIB_LDFLAGS])
